### ISSUE TITLE
[zydis] No package configuration file & linker error

### ISSUE THREAD

**Issue by VelocityRa** [microsoft/vcpkg#13547]
**Failure logs** 
```
1> [CMake] CMake Error at C:/vcpkg/scripts/buildsystems/vcpkg.cmake:493 (_find_package):
1> [CMake]   Could not find a package configuration file provided by "zydis" with any of
1> [CMake]   the following names:
1> [CMake] 
1> [CMake]     zydisConfig.cmake
1> [CMake]     zydis-config.cmake
```
etc

**To Reproduce**
`./vcpkg install zydis`
in a CMakeLists.txt: `find_package(zydis CONFIG REQUIRED)`  -> error
same for "Zydis", or without the `CONFIG`

**Additional context**
- I'm using manifests, so instead of the above i have `zydis` in my manifest and installed in the `vcpkg_installed` folder. But there are no `*config.cmake` files in its `vcpkg_installed/x64-windows/share/` dir.
- If I just `target_link_libraries` it without the `find_package` it can't find the `.lib`(s) at link time (though they do exist in `vcpkg_installed/x64-windows/lib/`).
- I have many other working deps in my proj, it's the port's fault I think.
- `link_directories("vcpkg_installed/x64-windows/lib")` + the `target_link_libraries` in the cmake as a hack, works.

**Host Environment**
- OS: Windows 10 x64
- Compiler: MSVC 16.7.0

**Comment by VelocityRa** [comment]
A config was added in https://github.com/zyantific/zydis/pull/138, however this was after the most recent release (which the vcpkg port is already at.

So, how are we supposed to use this port without it?

**Comment by sfhacker** [comment]
To be honest, I don't follow what the issue is here? Should I use a specific zydis version?
Just downloaded master branch and built from source. No issues at all.

![image](https://user-images.githubusercontent.com/25817529/93254200-525dd100-f7db-11ea-86d1-7068908ae3f4.png)

![image](https://user-images.githubusercontent.com/25817529/93254312-833e0600-f7db-11ea-97f0-0e0f184213a1.png)

I'm trying with version 3.1.0 now.

**Comment by sfhacker** [comment]
![image](https://user-images.githubusercontent.com/25817529/93254857-60f8b800-f7dc-11ea-988d-5b5b65dcb8c3.png)

There seems not to be a *config.cmake file in this particular version.
Perhaps you can generate one for this particular version or use .pc file.

Which package or port are you trying to build which needs zydis?

**Comment by VelocityRa** [comment]
@sfhacker I want to use zydis for my own C++ project.

Sorry, I realize now that the title is wrong. It's not a "build failure" of the port, but of vcpkg + my project that includes it (because CMake can't find it).

**Comment by sfhacker** [comment]
What's wrong with zydis master branch?

**Comment by VelocityRa** [comment]
I just mentioned the thing about the release since all ports I've seen are against specific library releases, not random commits on master.
Anyway, not sure how this relates to my issue. As it stands, vcpkg master has the issue I posted (unless I'm supposed to add it in CMake differently).

**Comment by athre0z** [comment]
The way I understand it, all you folks need is a new Zydis release that include the changes from https://github.com/zyantific/zydis/pull/138? That'd be no problem, we can make one -- please just confirm so it doesn't later turn out that we'd need additional changes.

**Comment by VelocityRa** [comment]
@athre0z Assuming vcpkg works with just that commit, yeah. I haven't tested this myself.

**Comment by NancyLi1013** [comment]
We will update to the next release once it is ready.

**Comment by NancyLi1013** [comment]
Hi @VelocityRa 

This should be fixed via PR #21490.
